ROLES.md

```diff
@@ -58,22 +57,55 @@ table describes:
     </td>
     <td>Highly experienced and active reviewer and contributor to an area</td>
     <td>Entry in one or more OWNERS files in GitHub, and write permissions
-        on one or more repos allowing PRs to be merged.
+        on one or more repos allowing PRs to be merged
     </td>
     <td>GitHub Directory</td>
   </tr>
 
   <tr>
-    <td>Lead</td>
+    <td>Working Group Technical Lead</td>
     <td>
         <p>Set priorities for a functional area and approve proposals</p>
         <p>Triage incoming issues, set milestones, repo labels</p>
-        <p>Run their working group</p>
+        <p>Roadmap alignment with top-level backlog</p>
+        <p>Mentor new contributors, project members, and approvers</p>
+        <p>Succession - identifying next steps for members of the working group</p>
+        <p>Responsible for technical health of their functional area</p>
```

Comment:
I would expect technical review of feature tracks to be done by this role.

Reply:
I added this below